modeling/gbert_classifier.py

```diff
@@ -0,0 +1,297 @@
+# -*- coding: utf-8 -*-
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.1
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # German BERT - minimal setup #22 
+# Issue link: https://github.com/dominikmn/one-million-posts/issues/22
+
+# %% [markdown]
+# The pre-trained model can be found under:
+# https://huggingface.co/deepset/gbert-base
+
```

Comment:
I would also include the tutorial with a comment like "Based on [tutorial](link)"